contrib/kind.sh

```diff
@@ -247,10 +247,12 @@ docker build -t ovn-daemonset-f:dev -f Dockerfile.fedora .
   --k8s-apiserver=https://[${API_IP}]:11337 \
   --ovn-master-count=${KIND_NUM_MASTER} \
   --kind \
-  --master-loglevel=5
+  --master-loglevel=5 \
+  --egress-firewall-enable=true
```

Comment:
Suggestion for consistency --> How about adding `EGRESS_FIREWALL_ENABLE=${EGRESS_FIREWALL_ENABLE:-true}` below L126 and modify L251 to be  `--egress-firewall-enable=${EGRESS_FIREWALL_ENABLE}

Reply:
We are no longer having an egress-firewall-enable flag it was deemed that this was lightweight enough in the absence of egressFirewall objects not to justify a flag